src/applications/personalization/profile/components/paperless-delivery/PaperlessDelivery.jsx

```diff
@@ -31,25 +34,24 @@ export const PaperlessDelivery = () => {
   const hasVAPServiceError = useSelector(hasVAPServiceConnectionError);
   const { loadingStatus } = communicationPreferencesState;
   const hasLoadingError = loadingStatus === LOADING_STATES.error;
-  const hasAPIError = hasVAPServiceError || hasLoadingError;
   const isLoading =
     loadingStatus === LOADING_STATES.idle ||
     loadingStatus === LOADING_STATES.pending;
-  const fetchCommunicationPreferences =
-    !hasAPIError && loadingStatus === LOADING_STATES.idle;
```

Comment:
Removed check for `LOADING_STATES.idle`. This means a network request to fetch communication preferences is made each time the user lands on the Paperless delivery. We want this behavior in order to reset the statuses of the checkboxes if the user navigates away from our page and back to it.